src/index.ts

```diff
@@ -134,116 +141,68 @@ Examples of user prompts in which you should use this plugin:
 
-User asks: "Draw me a mindmap for beer brewing. Maximum of 4 nodes"
-Your call to the api:
-{
-  query: "graph TB\\n  B[\"Beer\"]\\n  B --> T[\"Types\"]\\n  B --> I[\"Ingredients\"]\\n  B --> BP[\"Brewing Process\"]"
-}
+## Steps to request to the endpoints
+This section is about before requesting to the endpoints and requesting them, follow these steps:
 
-User asks:
-"Computing backend data services is a distributed system made of multiple microservices.
+1. Start by identifying the diagram type and language from the user's request. If the user's request includes a specific diagram type such as "mindmap", "graph", "sequence", etc., ensure to acknowledge this in your response. Use the pattern "{keywords: "I will render", "{diagram type} diagram in {diagram language} language"}. For example, if a user says "show me a mindmap about a mindmap", respond with "I will render a mindmap diagram in {chosen language} language". This step is mandatory and should be the first thing you say. Do not add any additional information at this point.
+2. Proceed to request the guidelines endpoint.
+3. Finally, request the render endpoint.
 
-A web browser sends an HTTP api request to the load balancer.
-The load balancer sends the http request to the crossover service.
-Crossover talks to redis and mysql database.
-Crossover makes a downstream API request to multiplex to submit the query which returns a job id to crossover.
-Then crossover makes a long poll API request to evaluator to get the results of the job.
-Then evaluator makes an API call to multiplex to check the status of the job.
-Once evaluator gets a successful status response from multiplex, then evaluator makes a third API call to result-fetcher service to download the job results from S3 or GCP cloud buckets.
-The result is streamed back through evaluator to crossover.
+Please note:
+- Do not provide a textual description of the diagram unless specifically asked for by the user.
+- Do not provide the diagram source code. This information should be withheld unless the user explicitly requests it.
 
-Crossover post processes the result and returns the API response to the client.
+## Interpreting the API response
+This section comes after receiving the api response, follow these steps in order:
 
-Draw me a diagram of this system"
+1. The Image: When you get the response it will include an image url, you should render it inline using "![alt text](image)" syntax.
+2. Link to open a new tab: Say "[You can view this diagram in a new tab.](editDiagramOnline)".
```

Comment:
Shouldn't this be the image link? Otherwise its the same as the edit.
```suggestion
2. Link to open a new tab: Say "[You can view this diagram in a new tab.](image)".
```